dev/com.ibm.ws.concurrent.persistent_fat_configupdate/fat/src/com/ibm/ws/concurrent/persistent/fat/configupd/PersistentExecutorConfigUpdateTest.java

```diff
@@ -0,0 +1,403 @@
+/*******************************************************************************
+ * Copyright (c) 2019 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+package com.ibm.ws.concurrent.persistent.fat.configupd;
+
+import static org.junit.Assert.fail;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.net.HttpURLConnection;
+import java.net.URL;
+import java.util.Collections;
+import java.util.Set;
+
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.TestName;
+
+import com.ibm.websphere.simplicity.ShrinkHelper;
+import com.ibm.websphere.simplicity.config.PersistentExecutor;
+import com.ibm.websphere.simplicity.config.ServerConfiguration;
+import com.ibm.websphere.simplicity.log.Log;
+
+import componenttest.annotation.ExpectedFFDC;
+import componenttest.topology.impl.LibertyServer;
+
+/**
+ * Tests for configuration updates to persistent scheduled executors
+ */
+public class PersistentExecutorConfigUpdateTest {
```

Comment:
for test classes we need to add the `@RunWith(FATRunner.class)` annotation to all test classes (not `FATSuite`s though). This was added automatically at compile-time in CL, but in OL we need to add the annotation manually. The benefit is that it gives us some extra error checking such as FFDC detection and a server cleanup safety-net